lib/addons/vpc-cni/index.ts

```diff
@@ -292,7 +299,7 @@ export interface CustomNetworkingConfig {
 
 const defaultProps: CoreAddOnProps = {
   addOnName: 'vpc-cni',
-  version: 'v1.13.4-eksbuild.1',
+  version: 'v1.14.1-eksbuild.1',
```

Comment:
Based on https://docs.aws.amazon.com/eks/latest/userguide/managing-vpc-cni.html the version should be v1.15.0-eksbuild.2 , however incremental update is needed. 

However, it looks like the network policy will not be available with this version, is that correct? If so, can we add a check to fail if version is less than 1.15 so that we don't get questions from the customers on why my network policy dos not work?

The error should state that customers should apply incremental update and use network policy starting from 1.15. 

I will need to include it in the release notes, so if you can update the PR dsc with this instructions as well that would help.